test/eChronicler/e_chronicler/journal_entry_test.exs

```diff
@@ -14,4 +14,18 @@ defmodule EChronicler.EChronicler.JournalEntryTest  do
     assert JournalEntry.format_datetime(incorrect_datetime)  == "no date found"
   end
 
+
+  test "truncate_journal_entry/1 truncates entry and adds ellipsis" do
+    assert JournalEntry.truncate_journal_entry_text("Time is an illusion. Teatime doubly so. And time for tennis? Forget about it.") == "Time is an illusion. Teatime doubly so. And time fo..."
+  end
+
+  test "truncate_journal_entry/1 doesn't add ellipsis after short entry" do
+    assert JournalEntry.truncate_journal_entry_text("Test.") == "Test."
+  end
+
+  test "truncate_journal_entry/1 returns blank entry" do
+    assert JournalEntry.truncate_journal_entry_text("") == ""
+  end
+
+
```

Comment:
Technically you want to include some unit tests for `create_journal_entry` here as well. But I think you'll probably get around to it in a future card, so not a blocker to merge. Just make a note that you want to include tests for it when you work on that feature.